be/src/exprs/vectorized/runtime_filter_bank.cpp

```diff
@@ -256,22 +257,31 @@ void RuntimeFilterProbeCollector::do_evaluate(vectorized::Chunk* chunk, RuntimeB
         return;
     }
     if (!eval_context.selectivity.empty()) {
+        const auto num_rows = chunk->num_rows();
+        auto& selection = eval_context.running_context.selection;
+        selection.assign(num_rows, 1);
+        auto has_filtered = false;
         for (auto& kv : eval_context.selectivity) {
             RuntimeFilterProbeDescriptor* rf_desc = kv.second;
             const JoinRuntimeFilter* filter = rf_desc->runtime_filter();
-            if (filter == nullptr) continue;
+            if (filter == nullptr) {
+                continue;
+            }
             ColumnPtr column = rf_desc->probe_expr_ctx()->evaluate(chunk);
-            vectorized::Column::Filter& selection = filter->evaluate(column.get(), &eval_context.running_context);
+            auto true_count = filter->evaluate(column.get(), &eval_context.running_context);
```

Comment:
true_count could be cumulative for multi filters. then we have more chance to early return if true_count ==0, and we needn't has_filtered flag.

Reply:
ok